packages/dd-trace/src/opentracing/span.js

```diff
@@ -39,13 +38,16 @@ class DatadogSpan extends Span {
 
   toString () {
     const spanContext = this.context()
+    const resource = spanContext._tags['resource.name'].length > 100
+      ? `${spanContext._tags['resource.name'].substring(0, 97)}...`
+      : spanContext._tags['resource.name']
```

Comment:
can you do a `const resourceName = spanContext._tags['resource.name']` and simplify all this? it's pretty hectic right now.